scripts/cat/cats.py

```diff
@@ -902,7 +895,7 @@ def status_change(self, new_status, resort=False):
     def rank_change_traits_skill(self, mentor):
         """Updates trait and skill upon ceremony"""
 
-        if self.status in ["warrior", "medicine cat", "mediator"]:
+        if self.status.is_working_any():
```

Comment:
Technically not "like for like" since the `is_working_any` check includes apprentices.